pkg/kvstore/etcd.go

```diff
@@ -479,6 +496,7 @@ func (e *etcdClient) renewSession(ctx context.Context) error {
 	log.Infof("Got new lease ID %x", newSession.Lease())
 
 	e.session = newSession
+	e.sessionCancel = sessionCancel
```

Comment:
The upstream commit here sends `sessionSuccess <- true` a few lines above. Was this intentionally omitted?

Reply:
This was not intentional and was an oversight. I'll fix it up.

Reply:
https://github.com/cilium/cilium/pull/12753